src/config.cc

```diff
@@ -288,6 +282,8 @@ void Config::initFieldCallback() {
       {"rocksdb.max_background_compactions", set_db_option_cb},
       {"rocksdb.max_background_flushes", set_db_option_cb},
       {"rocksdb.compaction_readahead_size", set_db_option_cb},
+      {"rocksdb.level0_slowdown_writes_trigger", set_db_option_cb},
```

Comment:
level0_slowdown_writes_trigger and level0_stop_writes_trigger is a column family option, can not use setdboption

Reply:
oh, thanks, i found unit test failed